tests/test_events.py

```diff
@@ -1,4 +1,4 @@
-from unittest import skip
+# from unittest import skip
```

Comment:
Delete this line (and the following, blank, line). Generally don't leave commented-out code in a file, unless it's in a transitional state.